packages/core/docs/guide/user-profile.md

```diff
@@ -0,0 +1,203 @@
+# User profile
+
+<!-- TODO: Add links to 'Composition API' and 'Composables' pages when they are ready -->
+
+> This page assumes you're familiar with Composition API and Composables. Read them first if you are new to these concepts.
+
+[[toc]]
+
+## Checking if the user is logged in
+
+Many interactions in the application are only available or are different if the customer is logged in. To check that, we will use [useUser](../composables/use-user.md), which is one of, if not the most, widely used composable in Vue Storefront.
+
+As with all composables, it's important to remember to call `load` before accessing any other property or function of `useUser`. Otherwise, `isAuthenticated` will always return `false`.
```

Comment:
```suggestion
Like with all other composables, it's important to remember to call `load` before accessing any other property or function of `useUser`. Otherwise, `isAuthenticated` will always return `false`.
```